config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "dish/:id", to: "dish#show"
+  patch "dish/:id", to: "dish#update"
```

Comment:
As mentioned above, a more RESTful option would be `post "/dishes/dish_id/ingredients", to: "dish_ingredients#create"`